cadquery/occ_impl/exporters/svg.py

```diff
@@ -176,11 +179,19 @@ def getSVG(shape, opts=None):
     strokeColor = tuple(d["strokeColor"])
     hiddenColor = tuple(d["hiddenColor"])
     showHidden = bool(d["showHidden"])
+    focus = None
+    if d["focus"] is not None:
+        focus = float(d["focus"])
 
     hlr = HLRBRep_Algo()
     hlr.Add(shape.wrapped)
 
-    projector = HLRAlgo_Projector(gp_Ax2(gp_Pnt(), gp_Dir(*projectionDir)))
+    coordinateSystem = gp_Ax2(gp_Pnt(), gp_Dir(*projectionDir))
```

Comment:
I haven't seen usage of camelCase for local variables in recent CQ commits.  Certainly there is usage in cq.py.  More recently names with underscore have been defined, for example:    

https://github.com/CadQuery/cadquery/blob/259cc7b4e47cf8da695085ab507963b57c60e046/cadquery/occ_impl/solver.py#L666

Names such as marginLeft in this function are an exception because they correspond to the option dictionary keys.  Perhaps that is another reason not to use camelCase names to avoid confusion with SVG option names.

Reply:
Done